OWNERS

```diff
@@ -2,20 +2,20 @@
 
 reviewers:
 - BenTheElder # lead
-- spiffxp # lead
-- stevekuznetsov # lead
+- spiffxp # chair
+- stevekuznetsov # chair
 - chaodaiG # oncall
-- cjwagner # oncall
+- cjwagner # oncall, lead
 - mpherman2 # oncall
-- fejta # oncall
 approvers:
```

Comment:
TODO: if we're adding people here based on chair/lead status still, @alvaroaleman is missing.